packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -263,393 +231,383 @@ class AccountSwitcher extends React.Component {
-                <div className='acc-switcher__footer'>
-                    {this.props.is_uk && this.props.has_any_real_account && (
-                        <Button
-                            className='acc-switcher__compare'
-                            type='button'
-                            has_effect
-                            onClick={this.showAccountTypesModal}
-                            text={localize('Compare')}
-                            secondary
-                        />
-                    )}
-                    <div id='dt_logout_button' className='acc-switcher__logout' onClick={this.handleLogout}>
-                        <Text color='prominent' size='xs' align='left' className='acc-switcher__logout-text'>
-                            {localize('Log out')}
-                        </Text>
-                        <Icon
-                            icon='IcLogout'
-                            className='acc-switcher__logout-icon drawer__icon'
-                            onClick={this.handleLogout}
-                        />
-                    </div>
+                <Text size='xs' color='prominent' className='acc-switcher__balance'>
+                    <Money
+                        currency={isRealAccountTab ? props.obj_total_balance.currency : 'USD'}
+                        amount={formatMoney(
+                            isRealAccountTab ? props.obj_total_balance.currency : 'USD',
+                            isRealAccountTab ? getTotalRealAssets() : getTotalDemoAssets(),
+                            true
+                        )}
+                        show_currency
+                        should_format={false}
+                    />
+                </Text>
+            </div>
+            <Text color='less-prominent' line_height='xs' size='xxxs' className='acc-switcher__total-subtitle'>
+                {total_assets_message}
+            </Text>
+            <div className='acc-switcher__separator' />
+            <div className='acc-switcher__footer'>
+                {props.is_uk && props.has_any_real_account && (
+                    <Button
+                        className='acc-switcher__compare'
+                        type='button'
+                        has_effect
+                        onClick={showAccountTypesModal}
+                        text={localize('Compare')}
+                        secondary
+                    />
+                )}
+                <div id='dt_logout_button' className='acc-switcher__logout' onClick={handleLogout}>
+                    <Text color='prominent' size='xs' align='left' className='acc-switcher__logout-text'>
+                        {localize('Log out')}
+                    </Text>
+                    <Icon icon='IcLogout' className='acc-switcher__logout-icon drawer__icon' onClick={handleLogout} />
                 </div>
             </div>
-        );
-    }
-}
+        </div>
+    );
+};
 
```

Comment:
Also add `is_fully_authenticated` to `propTypes` and to the `connect` to `client-store`:

`AccountSwitcher.propTypes = {
is_fully_authenticated: PropTypes.bool,
.... };`

`
const account_switcher = WithRouter(...
is_fully_authenticated: client.is_fully_authenticated,
...
`